control/injector/Injector.php

```diff
@@ -594,6 +594,20 @@ public function inject($object, $asType=null) {
 			}
 		}
 
+		// Populate named methods
+		if (isset($this->specs[$objtype]['methods'])) {
+			foreach ($this->specs[$objtype]['methods'] as $method) {
+				// Ignore any blank entries from the array; these may be left in due to config system limitations
+				if(!$method) continue;
+
+				list($methodName, $methodArgs) = $method;
+				call_user_func_array(
+					array($object, $methodName),
```

Comment:
This assumes the method is public. Are we happy with the default error that will be generated?